rosdep/python.yaml

```
@@ -1843,6 +1843,13 @@ python-graphviz-pip:
 python-gridfs:
   debian: [python-gridfs]
   fedora: [python-pymongo-gridfs]
+python-grip:
+  debian: [grip]
```

Arg, this is annoying.  Looking at https://packages.debian.org/search?keywords=grip&searchon=names&suite=all&section=all , it looks like `grip` is available in debian `buster` (the last release) and `sid` (the development release), but not `bullseye` (the current release) or `bookworm` (the release happening later this year).

In that case, we are going to have to have a more complicated key for debian:

```suggestion
  debian:
    '*': [grip]
    bookworm: null
    bullseye: null
    buster: [grip]
```